jwt.go

```diff
@@ -17,11 +17,20 @@ import (
 	"crypto/rsa"
 	jwt "github.com/dgrijalva/jwt-go"
```

Comment:
missed that in my PR, can you add an empty line between stdlib imports and 3rd party ones?